client/components/deposits-overview/recent-deposits-list.tsx

```diff
@@ -66,6 +68,39 @@ const RecentDepositsList: React.FC< RecentDepositsProps > = ( {
 	if ( isLoading || recentDeposits.deposits.length === 0 ) {
 		return <></>;
 	}
+
+	// Add a notice indicating the potential business day delay for pending and in_transit deposits.
+	let bannerAdded = false;
+	const depositRows = recentDeposits.deposits
+		.reverse() // reverse the array so that the oldest pending or in_transit deposit has the notice added to it.
+		.map( ( deposit ) => {
+			let bannerNotice = null;
+
+			if (
+				! bannerAdded &&
+				( 'pending' === deposit.status ||
+					'in_transit' === deposit.status )
+			) {
+				bannerNotice = (
+					<BannerNotice
+						status="info"
+						icon={ <InfoOutlineIcon /> }
+						children={ strings.notices.businessDayDelay }
+						isDismissible={ false }
+					/>
+				);
+				bannerAdded = true;
+			}
+
+			return (
+				<>
+					<DepositTableRow deposit={ deposit } />
+					{ bannerNotice }
+				</>
+			);
+		} )
```

Comment:
Since this is returning an array of elements, we need to add a unique `key` prop. ([docs](https://react.dev/reference/react/Fragment#rendering-a-list-of-fragments)). You will see an error in the browser console: `Each child in a list should have a unique "key" prop.`.


```suggestion
		return (
			<Fragment key={ deposit.id }>
				<DepositTableRow deposit={ deposit } />
				{ bannerNotice }
			</Fragment>
		);
```

Note: this error existed before your changes, I've also fixed this in #5822.

Reply:
Thanks :)

fixed in [491a673](https://github.com/Automattic/woocommerce-payments/pull/5839/commits/491a673cf27527672db45884099c7f63f68f2199).